### DISCUSSION TITLE
DLC in btcTransmuter

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#3096] [Ideas / Feature Requests]
Can discreet log contracts be used in [btcTransmuter](https://github.com/btcpayserver/btcTransmuter)?

One example: The [crowdfunding](https://docs.btcpayserver.org/Apps/#crowdfunding-app) part of BTCPay can use DLC so that a part of funds are locked until the project is finished, used in other projects that use DLC like https://atomic.finance/ and funds are released once project is finished based on terms that everyone agreed to.

This will improve crowdfunding feature, create more incentives for everyone to use it and everything is done using bitcoin technologies.

Can even add other things later like others betting if a project will be finished on time or not.

This will require [NDLC](https://github.com/dgarage/NDLC) library and I am not sure if it's in sync with all the progress happening in DLC specs and [bitcoin-s](https://github.com/bitcoin-s/bitcoin-s)

**Comment by pavlenex** [comment]
No, transmuter is something we'll probably discontinue in the long-run so I don't think any such features will be planned.

**Comment by ghost** [comment]
Any reasons to discontinue it? There are few people who prefer to trigger lot of things based on payments received in BTCPay. One question from Stackexchange that I remember was someone looking to exchange BTC to stablecoins regularly with automation. What will be the alternatives for such users?

**Reply by ghost** [reply]
Plugins are the future. 
iirc there's the idea to transmute the transmuter to a plugin. :wink:

**Reply by pavlenex** [reply]
Maintenance burden, not enough users use it due to its complexity.
I don't thin transmuter will ever be a plugin in this shape and form, but some basics functionality of it could be implemented in the UI. There's no ETA for it though.
